LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/IoTHubImpl/IoTHubDeviceTwin.cs

```diff
@@ -10,13 +10,12 @@ public class IoTHubDeviceTwin : IDeviceTwin
     {
         internal Twin TwinInstance { get; }
 
-        public TwinProperties Properties => this.TwinInstance.Properties;
+        public ITwinPropertiesContainer Properties { get; }
 
         public IoTHubDeviceTwin(Twin twin)
         {
-            ArgumentNullException.ThrowIfNull(twin, nameof(twin));
```

Comment:
Is there a reason to remove the null check ? Is it performed somewhere else?

Reply:
Probably not, I re-added this check.